functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -407,13 +409,45 @@ def add_prediction_labels(self, prediction_labels: list, training_id: int):
                 #TODO: Update some sort of training status table?
                 #self._update_training_status(training_id,conn)
                 conn.commit()
-            # logging.debug('Inserted {0} predictions for training session {1}'.format(labels_length, training_id))
             finally: cursor.close()
+            logging.debug('Inserted {0} predictions for training id {1}'.format(labels_length, training_id))
         except Exception as e:
             logging.error("An errors occured updating tagged image: {0}".format(e))
             raise
         finally: conn.close()
 
+    def add_training_session(self, training: TrainingSession, user_id: int):
+        training_id = -1
+        try:
+            conn = self._db_provider.get_connection()
+            try:
+                cursor = conn.cursor()
+                query = ("WITH t AS ( "
+                        "INSERT INTO Training_Info (TrainingDescription, ModelLocation, ClassPerfAvg, CreatedByUser) "
+                        "VALUES ({},{},{},{}) RETURNING TrainingId), "
+                        "p AS (INSERT INTO Class_Performance (TrainingId,ClassificationId,AvgPerf) "
+                            "VALUES ")
+                query.format(training.description,training.model_url,training.avg_perf,user_id)
+
+                num_of_classes = len(training.class_perf)
```

Comment:
It makes sense to me but a comment explaining what's happening here will be great

Reply:
@samiyaakhtar Sure. Will add something